fips/models.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import unicode_literals
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from calaccess_raw.models.base import CalAccessBaseModel
+
+
+@python_2_unicode_compatible
+class FipsEntity(CalAccessBaseModel):
```

Comment:
Not sure it's necessary to extend from `calAccessBaseModel`, the primary reason to do that would be if we're bulk loading from csv. Not sure if theres any other reason.